dao/dynamodb.rya/src/main/java/mvm/rya/dynamodb/dao/DynamoRdfConfiguration.java

```diff
@@ -0,0 +1,83 @@
+package mvm.rya.dynamodb.dao;
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.hadoop.conf.Configuration;
+
+import mvm.rya.api.RdfCloudTripleStoreConfiguration;
+
+public class DynamoRdfConfiguration extends RdfCloudTripleStoreConfiguration{
+	
+	public static final String USE_MOCK = ".useMockInstance";
+	public static final String AWS_USER = "rya.aws.user";
+	public static final String AWS_SECRET_KEY = "rya.aws.secret.key";
+	public static final String AWS_ENDPOINT = "rya.aws.endpoint";
+	public static final String DYNAMO_TABLE_NAME = "rya.dynamo.tablename";
```

Comment:
The table name here is kind of synonymous with the mongo collection or the rya table prefix in accumulo.